os-eap7-launch/added/launch/messaging.sh

```diff
@@ -172,6 +173,7 @@ function generate_resource_adapter() {
                     <config-property name=\"ServerUrl\">tcp://$6:$7?jms.rmIdFromConnectionId=true</config-property>
                     <connection-definitions>
                         <connection-definition
+                              "${13}"
```

Comment:
this is already inside quotes , did you mean just ${13} ?

Reply:
Its ok use quotes there since it is not escaped.